python/ray/serve/utils.py

```diff
@@ -378,7 +378,10 @@ def import_attr(full_path: str):
 
 
 async def mock_imported_function(batch):
-    return [await request.body() for request in batch]
+    result = []
```

Comment:
lint

Reply:
OOC why does this not work on BuildKite?